webapp/channels/src/utils/emoticons.tsx

```diff
@@ -50,6 +50,7 @@ export function matchEmoticons(text: string): string[] | null {
 export function handleEmoticons(
     text: string,
     tokens: Map<string, {value: string; originalText: string}>,
+    renderEmoticonsAsEmoji?: boolean,
```

Comment:
Could you change this so that the default behaviour is to render emotions as emojis? The `DEFAULT_OPTIONS` in `text_formatting.tsx` only applies when there's no options passed at all, so if we want this to default to true, we have to handle the case where an options object is passed but `renderEmoticonsAsEmoji` isn't passed

```suggestion
    renderEmoticonsAsEmoji = true,
```